packages/types/src/sdk/search.ts

```diff
@@ -54,7 +54,7 @@ export interface SearchFilters {
     [key: string]: any[]
   }
   [SearchFilterOperator.CONTAINS]?: {
-    [key: string]: any[]
+    [key: string]: any[] | any
```

Comment:
Everything else in this type takes an array of any, and now this is the only one that takes an array of any OR any, and that makes me feel a bit uneasy.

Reply:
Is this because what the frontend sends? Could we not tweak the frontend instead?

Reply:
Is this because what the frontend sends? Could we not tweak the frontend instead?

Reply:
> Is this because what the frontend sends? Could we not tweak the frontend instead?

Sadly if the frontend has been sending something which isn't an array, we can't really change that, as these searches will be baked into apps, it isn't super easy to update them all (since client libraries aren't updated with the server version right now).

I think rather than updating our typing around this, could we instead update the API controller to look over the search operators and re-jig any which won't fit into our expected structure? That way we could keep the typing clean and just have the fact that the frontend doesn't consistently send good search parameters hidden in the API layer?

Reply:
Done! Much nicer. Thanks folks 🙂